smart_open/smart_open_lib.py

```diff
@@ -387,6 +387,34 @@ def _patch_pathlib(func):
     return old_impl
 
 
+def smart_open(
+        uri,
+        mode='rb',
+        buffering=-1,
+        encoding=None,
+        errors=None,
+        newline=None,
+        closefd=True,
+        opener=None,
+        ignore_ext=False,
+        **kwargs,
+        ):
```

Comment:
De-dent the closing `):` please (closing bracket one less indent level than body).